### PR TITLE
Necropolis IPCs

### PR DESCRIPTION
**Pull request by ParadoxSpace**
As on the tin.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

**Comment by NonQueueingMatt** [comment]
There are no complaints about IPCs being a liaison?

**Comment by Arrow768** [comment]
Currently waiting on lore to get back to me regarding a yes/no for IPC liaisons.

**Comment by Arrow768** [comment]
DNM until the wiki warning is added.

File: code/game/jobs/faction/zavodskoi.dm [hunk 1]
```diff
@@ -47,6 +51,7 @@
 		/datum/species/skrell,
 		/datum/species/unathi,
 		/datum/species/diona
```
Comment (Alberyk) [inline code/game/jobs/faction/zavodskoi.dm:53]:
```suggestion
		/datum/species/diona,
```

**Pull request merged**